x-pack/plugins/apm/server/routes/storage_explorer/get_service_statistics.ts

```diff
@@ -233,7 +233,7 @@ export async function getServiceStatistics({
               sampledTransactionDocs / totalTransactionsPerService[serviceName],
               1
             )
-          : 0;
+          : 'N/A';
```

Comment:
I think it might be better to return `null` here. "N/A" is a presentational and locale-specific. It might be slightly more code if we returned `null` here and made the UI display "N/A" or the appropriate string.

Let me know if I can help in any way with getting this done.

Reply:
cc @miltonhultgren do we have anything in our API guidelines about handling 0 vs. null/undefined.

Reply:
I agree that the API should not return a (localized) string in this situation (based on the API guidelines to separate API from UI presentation), and if this is meant to express a lack of a value then I would return a null/undefined (but this part is not stated in the guidelines).

This is for the reason that 0 may be a valid value in some cases, so it's risky to make 0 === N/A, while null is usually the standard invalid value.